src/kibana/plugins/vis_types/vislib/editors/line.html

```diff
@@ -0,0 +1,7 @@
+<vislib-basic-options></vislib-basic-options>
+<div class="vis-option-item">
```

Comment:
With all the copy and paste, I think it makes sense to add this to the `vislib-basic-options` directive :point_up: